docs/modules/operation/pages/deep-dive/user-management/security-roles.adoc

```diff
@@ -2,10 +2,10 @@
 [[ga-role-user-management-roles]]
 = Assign User Permissions
 
-Create user permissions by assigning security roles.
+Create user permission levels by assigning security roles.
 These roles regulate access to the web UI and the REST API to exchange monitoring and inventory information.
 ifndef::opennms-prime[]
-In a distributed installation, the Minion instance requires the ROLE_MINION permission to interact with {page-component-title}.
+In a distributed installation, the Minion instance requires the `ROLE_MINION` permission to interact with {page-component-title}.
```

Comment:
```suggestion
In a distributed installation, Minion instances require an account with the `ROLE_MINION` permission to interact with {page-component-title}.
This can be one account for all Minions to share, or one account per Minion.
```